content/docs/forms.md

```diff
@@ -138,7 +138,7 @@ In HTML, `<select>` creates a drop-down list. For example, this HTML creates a d
 </select>
 ```
 
-Note that the Coconut option is initially selected, because of the `selected` attribute. React, instead of using this `selected` attribute, uses a `value` attribute on the root `select` tag. This is more convenient in a controlled component because you only need to update it in one place. For example:
+ध्यान दीजिये की `selected` एट्रिब्यूट की वजह से Coconut ऑप्शन पहले से सेलेक्टेड है। React में, `selected` एट्रिब्यूट इस्तेमाल करने के बजाए, `value` एट्रिब्यूट का इस्तेमाल रुट `select` टैग पर किया जाता है। यह एक कंट्रोल्ड कौम्पोनॅन्ट में बहुत सुविधाजनक रहता है क्यूंकि हमे इसे सिर्फ एक जगह अपडेट करना होता है। उदाहरण के तौर पर:
```

Comment:
React में, `selected` एट्रिब्यूट इस्तेमाल करने के बजाए, `value` एट्रिब्यूट का इस्तेमाल रुट `select` टैग पर किया जाता है। does not make sense grammatically

Reply:
Missed this one too

Reply:
This is still missing